src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -185,6 +185,7 @@ public byte Read(Register.Address registerAddress, Port port = Port.PortA, Bank
         /// <returns>The value of the register bit read.</returns>
         public bool ReadBit(Register.Address registerAddress, int bitNumber, Port port = Port.PortA, Bank bank = Bank.Bank1)
         {
+            ValidateBitNumber(bitNumber);
```

Comment:
I thought you also planned to remove the validation from the `GetBit` method so we don't end up validating twice

Reply:
Oops.  Thx